script/dev-deploy

```diff
@@ -95,6 +95,21 @@ def scp_command(local_file, remote, remote_file) -> str:
     return cmd
 
 
+def set_rm_intermediate(rm: bool) -> str:
+    return " --rm " if rm else ""
+
+
+def set_config_path(custom: str, origin: str) -> str:
+    return f" -v {custom}:{origin} "
+
+
+def set_resource_constraints(memory_cpus: tuple[str, str]) -> str:
+    memory, cpus = memory_cpus
+    return (f" --memory={memory} " if memory is not None else "") + (
+        f" --cpus={cpus} " if cpus is not None else ""
+    )
```

Comment:
Since memory & cpus can not be zero, we can simplify it with the following:

```
f" --memory={memory} " if memory else ""
```

Reply:
thx [ef21997](https://github.com/hstreamdb/hstream/pull/816/commits/ef21997e5d043165679b2d0f396b331891dbffbb)